wifiphisher/pywifiphisher.py

```diff
@@ -27,6 +27,7 @@
 import wifiphisher.common.accesspoint as accesspoint
 import wifiphisher.common.tui as tui
 import wifiphisher.common.opmode as opmode
+from .common.dependencies import check_dependencies
```

Comment:
You mean: `wifiphisher.common.dependencies`

Reply:
no that uses a relative path which is still valid.

Reply:
I know. But we should stick to one notation. Either comply with the rest or change all of them.